### PR TITLE
docs(README.md): adding more details about `docker-compose` cmd 

### PR DESCRIPTION
**Pull request by kamalbuilds**
## Description

README is missing important information about `docker-compose` cmd which can be very useful for new comers starting out with Fossology .

### Changes

Added details about  `docker-compose` cmd and refactored some code.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -53,12 +53,17 @@ docker run -p 8081:80 fossology/fossology
...
+
+If you want to run Fossology with an external database container, you can use Docker Compose, via the following command: 
 
 ```sh
 docker-compose up
 ```
 
+Docker Compose is a tool that allows you to define and run multi-container applications using a YAML file. FOSSology provides a docker-compose.yml file that defines three services: scheduler, web, and db. The scheduler service runs the FOSSology scheduler daemon, which handles the analysis tasks. The web service runs the FOSSology web server, which provides the web interface. The db service runs a PostgreSQL database server, which stores the FOSSology data. You can use the docker-compose up command to start all three services at once.
+
+The Fossology web service allows you to configure its database connection using some environment variables. These variables are defined in the docker-compose.yml file under the environment key.
```
Comment (GMishx) [inline README.md:65]: Please avoid writing long lines. It becomes very difficult to review them. Please add line breaks at every 80 characters or so.

Comment (GMishx) [inline README.md:65]:
```suggestion
The FOSSology web service allows you to configure its database connection using some environment variables. These variables are defined in the docker-compose.yml file under the environment key.
```

File: README.md [hunk 2]
```diff
@@ -53,12 +53,17 @@ docker run -p 8081:80 fossology/fossology
...
+
 The Docker image allows the configuration of its database connection over a set of environment variables.
```
Comment (GMishx) [inline README.md:67]: Both lines are conveying the same message, thus are repeating the sentence. Please refactor.

**Review by GMishx — CHANGES_REQUESTED** [review]
@legendarykamal Thank you for your contribution. Please follow our [contributing guidelines](https://github.com/fossology/fossology/blob/59c5d1c0b00e06c43a961464596fd702b12f8e95/CONTRIBUTING.md#git-commit-conventions) for the commit conventions.

Also, please squash your commits into 1.

**Comment by GMishx** [comment]
@legendarykamal Please update the **How to test** section of the PR or remove it.

**Comment by legendarykamal** [comment]
Thank you @GMishx for the feedback , have taken into account added some line breaks and some code refactorations.

Have squashed the commits too , thank you.

**Pull request merged**